spec/values/calculation/round/three_arguments.hrx

```diff
@@ -0,0 +1,204 @@
+a {b: round(nearest, 117px, 25px)}
+
+<===> strategy/nearest/positive/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> strategy/up/positive/input.scss
+a {b: round(up, 101px, 25px)}
+
+<===> strategy/up/positive/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> strategy/up/negative/input.scss
+a {b: round(up, -10, -100)}
+
+<===> strategy/up/negative/output.css
+a {
+  b: -100;
+}
+
+<===>
+================================================================================
+<===> strategy/up/positive_and_infinity/input.scss
+a {b: round(up, 10, infinity)}
+
+<===> strategy/up/positive_and_infinity/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> strategy/up/negative_and_infinity/input.scss
+@use "sass:math";
+a {b: math.div(1, round(up, -10, infinity))}
+
+<===> strategy/up/negative_and_infinity/output.css
+a {
+  b: calc(-infinity);
+}
+
+<===>
+================================================================================
+<===> strategy/up/infinity/input.scss
+@use "sass:math";
+a {b: round(up, infinity, infinity)}
+
+<===> strategy/up/infinity/output.css
+a {
+  b: calc(NaN);
+}
+
+<===>
+================================================================================
+<===> strategy/down/positive/input.scss
+a {b: round(down, 122px, 25px)}
+
+<===> strategy/down/positive/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> strategy/down/negative/input.scss
+a {b: round(down, -10, -100)}
+
+<===> strategy/down/negative/output.css
+a {
+  b: 0;
+}
+
+<===>
+================================================================================
+<===> strategy/down/infinity/input.scss
+a {b: round(down, infinity, infinity)}
+
+<===> strategy/down/infinity/output.css
+a {
+  b: calc(NaN);
+}
+
+<===>
+================================================================================
+<===> strategy/down/positive_and_infinity/input.scss
+@use "sass:math";
+a {b: math.div(1, round(down, 10, infinity))}
+
+<===> strategy/down/positive_and_infinity/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> strategy/down/negative_and_infinity/input.scss
+@use "sass:math";
+a {b: round(down, -10, infinity)}
+
+<===> strategy/down/negative_and_infinity/output.css
+a {
+  b: calc(-infinity);
+}
+
+<===>
+================================================================================
+<===> strategy/to-zero/positive/input.scss
+a {b: round(to-zero, 120px, 25px)}
+
+<===> strategy/to-zero/positive/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> strategy/to-zero/negative/input.scss
```

Comment:
All other edge cases are implicitly tested either in the two-argument file (since `nearest` is the default strategy), or in the `up` / `down` directory, since it's the same formula. (See file [here](https://github.com/sass/dart-sass/blob/8fdd53ebe2b4174e0e31d6c2bf6f98e834875dca/lib/src/value/calculation.dart#L709))

Reply:
I think it would be clearer to test `nearest` in the `three_arguments` directory and leave them implicit in `two_arguments` instead (with just enough to verify that `nearest` is in fact the strategy being used).

It's probably better to duplicate tests between `up`/`down` and `nearest`, just to be on the safe side.

Reply:
Got it! Just added the same tests for `nearest` in the three_argument directory